test/config/chaosduck.yaml

```diff
@@ -77,11 +77,6 @@ spec:
         # and substituted here.
         image: ko://knative.dev/pkg/leaderelection/chaosduck
 
-        args: [
-          # TODO(https://github.com/knative/eventing/issues/3590): Enable once IMC chaos issues are fixed.
-          "-disable=imc-controller", "-disable=imc-dispatcher",
```

Comment:
We will lose events if we kill the `imc-dispatcher`.

Reply:
I think we can disable HA in the IMC dispatcher reconciler.

Reply:
Or disable chaos for it.

Reply:
Yeah, just wanted to see what would happen and not surprised :) Trying now with just the control plane.